pages/repo/preview.js

```diff
@@ -0,0 +1,81 @@
+import React, { useRef } from 'react'
+import { withRouter } from 'next/router'
+import { graphql, compose } from 'react-apollo'
+import gql from 'graphql-tag'
+import { ColorContextProvider } from '@project-r/styleguide'
+import { renderMdast } from 'mdast-react-render'
+
+import Loader from '../../components/Loader'
+import Frame from '../../components/Frame'
+import { getSchema } from '../../components/Templates'
+
+import * as fragments from '../../lib/graphql/fragments'
+import initLocalStore from '../../lib/utils/localStorage'
+import withT from '../../lib/withT'
+
+const getCommitById = gql`
+  query getCommitById($repoId: ID!, $commitId: ID!) {
+    repo(id: $repoId) {
+      ...EditPageRepo
+      commit(id: $commitId) {
+        ...CommitWithDocument
+      }
+    }
+  }
+  ${fragments.EditPageRepo}
+  ${fragments.CommitWithDocument}
+`
+
+const PreviewPage = ({ dark, router, data = {} }) => {
+  const {
+    loading,
+    error,
+    repo: {
+      commit: { document }
+    }
+  } = data
```

Comment:
This is leads to an error with the new Publikator for docs that haven't been committed yet.
<img width="1273" alt="Screenshot 2021-03-10 at 17 20 57" src="https://user-images.githubusercontent.com/3907984/110661672-1a9d7800-81c5-11eb-8cb3-e5fe5b0b1afb.png">

Reply:
got it. fixed